bin/setup

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
+echo 'gem: --no-ri --no-rdoc --no-document' > ~/.gemrc
```

Comment:
This needs to stay in $CI, because we don't want to mess with local .gemrc files on personal machine.

Reply:
:+1: